internal/runtimehandlerhooks/runtime_handler_hooks.go

```diff
@@ -44,3 +54,23 @@ func highPerformanceAnnotationsSpecified(annotations map[string]string) bool {
 	}
 	return false
 }
+
+func cpuLoadBalancingAllowed(config *libconfig.Config) bool {
+	cpuLoadBalancingAllowedAnywhereOnce.Do(func() {
+		for _, runtime := range config.Runtimes {
+			for _, ann := range runtime.AllowedAnnotations {
+				if ann == crioann.CPULoadBalancingAnnotation {
+					cpuLoadBalancingAllowedAnywhere = true
```

Comment:
The reason I was questioning the name is that the annotation is only used to disable cpu balancing.